### DISCUSSION TITLE
Toggling fullscreen

### DISCUSSION THREAD

**Discussion by reksc** [rxhanson/Rectangle#1011] [General]
Hi, I googled the topic but feels like I'm quite alone with this one :D.

During my first minutes with Rectangle I moved the windows around with the shortcuts and as a natural consequence I asked myself "ok, how about fullscreen?".

The "Maximize" function adjusts the size of the window to screen width and height, that's logical.

However, I don't see anything related to toggling fullscreen within Rectangle.

Is this due to the existing, native shortcut (**cmd + ctrl + f**) ?

Edit: oh, and the icon used for maximize is a bit misleading - it's almost identical to fullscreen icon from the MacOs dropdown under the green bullet.

Many thanks for a great app anyway! :)

**Comment by rxhanson** [comment]
Thanks for the feedback. Rectangle intentionally does not overlap with macOS functionality, and won't provide keyboard shortcut mapping to existing keyboard shortcuts or macOS functionality since that would be outside the scope of the app. You can reconfigure the menu item shortcut for full screen in System Preferences, I'm sure there are tutorials on how to do this. 

Rectangle Pro (paid) does allow a way to enter full screen using the window throw or quick throw functionality.

Fullscreen on macOS will convert a window into a "Space", and macOS does not have a public API for using Spaces, and this is why I don't provide ways to manipulate spaces in Rectangle & Rectangle Pro.

**Reply by reksc** [reply]
Thanks for taking your time for a detailed response, much appreciated. I'm impressed how professionally you handle the project and its community.

I will just stick to the OS functionality, it's a perfect combo with Rectangle shortcuts.

All best!
